CMakeLists.txt

```diff
@@ -34,9 +34,11 @@ else (WIN32)
     endif()
 
 	IF (CMAKE_SYSTEM_PROCESSOR MATCHES "^(i.86|x86(_64)?)$")
-        if (RAIBLOCKS_SIMD_OPTIMIZATIONS)
-		    set (PLATFORM_COMPILE_FLAGS "${PLATFORM_COMPILE_FLAGS} -msse4")
-        endif()
+		if (RAIBLOCKS_SIMD_OPTIMIZATIONS)
+			set (PLATFORM_COMPILE_FLAGS "${PLATFORM_COMPILE_FLAGS} -msse4")
+		elseif(CMAKE_SYSTEM_PROCESSOR STREQUAL "x86_64")
+			set (PLATFORM_COMPILE_FLAGS "${PLATFORM_COMPILE_FLAGS} -msse2")
```

Comment:
Do we need to set -msse2 here if it's set already above?

Reply:
Where?